### DISCUSSION TITLE
include-hosts-custom.txt

### DISCUSSION THREAD

**Discussion by crubik** [xtrime-ru/antizapret-vpn-docker#15] [Q&A]
после обновления до 2.0.12 перестал открываться сайт ntc.party. тасировка идет через провайдера. ручками добавил этот сайт в include-hosts-custom и перезапустил докер - не помогло

**Comment by xtrime-ru** [comment]
А что выдает команда?
```
docker compose exec antizapret-vpn dig ntc.party @127.0.0.4
```

**Comment by crubik** [comment]
; <<>> DiG 9.18.28-0ubuntu0.24.04.1-Ubuntu <<>> ntc.party @127.0.0.4
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 25987
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;ntc.party.                     IN      A

;; ANSWER SECTION:
ntc.party.              300     IN      A       10.224.0.5

;; Query time: 28 msec
;; SERVER: 127.0.0.4#53(127.0.0.4) (UDP)
;; WHEN: Wed Aug 07 20:49:39 UTC 2024
;; MSG SIZE  rcvd: 54

**Reply by xtrime-ru** [reply]
ip 10.224.0.5 - все ок получается. 

А на компьютере, где проблема как резолвится?

**Comment by crubik** [comment]
странно. ожило

**Reply by xtrime-ru** [reply]
А сколько времени после старта контейнера прошло? Он же несколько минут стартует

**Reply by xtrime-ru** [reply]
У меня кстати и без include-hosts-custom ntc.party резолвится через vpn

**Comment by crubik** [comment]
всё ожило.
